common/io.hpp

```diff
@@ -170,11 +170,11 @@ class StringReader : virtual public Reader {
 		reader_ {new StreamReader(s_)} {
 	}
 	StringReader(StringReader &&sr) :
-		s_ {move(sr.s_)},
+		s_ {std::move(sr.s_)},
```

Comment:
Interesting that it warns for this specific call, but [their reasoning makes sense](https://reviews.llvm.org/D119670).